scripts/determine-version.js

```diff
@@ -118,11 +118,20 @@ async function main() {
   } else {
     // Dev
     const timestamp = await getCommitTimestamp();
-    const ts = `${timestamp.getUTCFullYear()}${
-      timestamp.getUTCMonth() + 1
-    }${timestamp.getUTCDate()}${timestamp.getUTCHours()}${
-      timestamp.getUTCMinutes() + 0
-    }${timestamp.getUTCSeconds()}`;
+    const formattedMonth = (timestamp.getUTCMonth() + 1)
+      .toString()
+      .padStart(2, "0");
+    const formattedDate = timestamp.getUTCDate().toString().padStart(2, "0");
+    const formattedHours = timestamp.getUTCHours().toString().padStart(2, "0");
+    const formattedMinutes = timestamp
+      .getUTCMinutes()
+      .toString()
+      .padStart(2, "0");
+    const formattedSeconds = timestamp
+      .getUTCSeconds()
+      .toString()
+      .padStart(2, "0");
+    const ts = `${timestamp.getUTCFullYear()}${formattedMonth}${formattedDate}${formattedHours}${formattedMinutes}${formattedSeconds}`;
```

Comment:
It's not part of libplanet's lint settings, but is it possible to adjust this code to fit within 80 characters per line?

Reply:
Done, let me know how it looks (501e624)

Reply:
> Done, let me know how it looks ([501e624](https://github.com/planetarium/libplanet/commit/501e6249f42d842a132897756d9fd2dd40b94858))

Thank you. However, when I save this code in an IDE like vscode, it reverts to the original code. So I think the following code is better, what do you think?

```js
    const ts =
      `${timestamp.getUTCFullYear()}` +
      `${formattedMonth}` +
      `${formattedDate}` +
      `${formattedHours}` +
      `${formattedMinutes}` +
      `${formattedSeconds}`;
```

Reply:
You're right. That also gives us room to do the formatting in-line, so we could remove the variable declarations:

```js
const timestamp = await getCommitTimestamp();
const ts =
  timestamp.getUTCFullYear() +
  (timestamp.getUTCMonth() + 1).toString().padStart(2, "0") +
  timestamp.getUTCDate().toString().padStart(2, "0") +
  timestamp.getUTCHours().toString().padStart(2, "0") +
  timestamp.getUTCMinutes().toString().padStart(2, "0") +
  timestamp.getUTCSeconds().toString().padStart(2, "0");
```